credentialdigger/download.py

```diff
@@ -30,18 +30,19 @@
 # https://github.com/explosion/spaCy/blob/master/spacy/cli/download.py
 
 
-def download(model, *pip_args):
+def download(args):
     """ Download a model and link it to the credental digger models_data
     folder.
 
     Parameters
     ----------
     model: str
         The name of the model. It must be an environment variable.
-    **pip_args
+    pip_args
         Keyword arguments for pip.
```

Comment:
What's the type of this parameter? Moreover, it's not **kwargs anymore so the description may need to be updated